make.go

```diff
@@ -279,6 +279,26 @@ func runGitCommandIn(dir string, arg ...string) error {
 	return cmd.Run()
 }
 
+func execGitConfig(args ...string) (string, error) {
+	gitArgs := append([]string{"config", "--get", "--null"}, args...)
```

Comment:
Can you add a comment as to why --null is required here?

Reply:
Yes, I will. Per `git-config` manpage, it "allows for secure parsing of the output without getting confused e.g. by values that contain line breaks." Thank you!